src/codegen/util/handlebars.js

```diff
@@ -27,11 +27,9 @@ handlebars.registerHelper('parameterList', (operation) =>
 handlebars.registerHelper('operationPath', (operation) =>
   serviceUtil.getEndpointUriStr(operation));
 
-handlebars.registerHelper('nonGetMethod', (operation, options) => {
-  if (operation.method !== 'GET') {
-    return options.fn(this);
-  }
-  return '';
+// eslint-disable-next-line func-names
```

Comment:
Handlebars helpers need to be able to reference `this` inside the function body. Using arrow functions won't work.